XamarinCommunityToolkit/Effects/EffectIds.shared.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Xamarin.Forms;
+
+[assembly: ResolutionGroupName("Xamarin.CommunityToolkit.Effects")]
+
+namespace Xamarin.CommunityToolkit.Effects
+{
+	sealed class EffectIds
+	{
+        static string EffectResolutionGroupName = $"{nameof(Xamarin)}.{nameof(CommunityToolkit)}.{nameof(Effects)}";
```

Comment:
I should not have commited this change, the field is private so should be camelCased, this also wan't changed in the rest of the class so I broke the build 🤪